osc/util/debquery.py

```diff
@@ -86,6 +72,40 @@ def read(self, all_tags=False, self_provides=True, *extra_tags):
         self.__parse_control(control, all_tags, self_provides, *extra_tags)
         return self
 
+    def __open_tar(self, arfile):
+        control = arfile.get_file(b'control.tar')
+        if control:
+            # XXX: python2.4 relies on a name
```

Comment:
This is probably not needed as we support python 3.6+
Maybe just remove the comment? Not sure if it is still relevant.

Reply:
I removed all of the python 2.x stuff for compatibility in395df87f.

This one referred to the `name=` argument that was always required for python 2.4 even when using `fileobj=`, but for later versions only the latter is required.